runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -552,3 +583,33 @@ func (c *Connection) estimateSize(ctx context.Context) (int64, error) {
 
 	return size, nil
 }
+
+// latestRCUPerService returns the sum latest RCU value reported for nodes in each service i.e. read/write.
+func (c *Connection) latestRCUPerService(ctx context.Context) (map[string]float64, error) {
+	var query string
+	if c.config.Cluster == "" {
+		query = "SELECT service as billing_service, anyLast(value) as latest_value from billing.events WHERE event_name = 'rcu' GROUP BY service"
+	} else {
+		query = fmt.Sprintf(`SELECT service as billing_service, sum(value) AS latest_value FROM (SELECT service, anyLast(value) as value FROM clusterAllReplicas('%s', billing.events) WHERE event_name = 'rcu' GROUP BY hostName(), service) GROUP BY billing_service`, c.config.Cluster)
```

Comment:
Is `billing.events` a Clickhouse system table? This code also runs for self-managed Clickhouses

Reply:
No its a table we create for our internal cluster, in case its not present it will just print a warning and do nothing. Should we just ignore printing error of move it to debug level?

Reply:
or another way is to have a config on connector like `rill_managed` or something like that, when set we will emit otherwise not.